Bottom: 4340bf1e842514dbbc7f7e3d2d67d5e812f4a035
Top:    25b370973988353131f3b8ea53234d10e60eda64
Author: Ian Kent <raven@themaw.net>
Date:   2013-08-19 11:07:20 +0800

Refresh of debug.patch

---

diff --git a/modules/mount_autofs.c b/modules/mount_autofs.c
index 3d96e69..8c1e600 100644
--- a/modules/mount_autofs.c
+++ b/modules/mount_autofs.c
@@ -184,7 +184,6 @@ int mount_mount(struct autofs_point *ap, const char *root, const char *name,
 		return 1;
 	}
 	argv[0] = info->map;
-	error(LOGOPT_ANY, "info type %s format %s map %s", info->type, info->format, info->map);
 
 	if (options) {
 		p = options;
